package.json

```diff
@@ -7,12 +7,13 @@
     "url": "https://github.com/webscopeio/react-textarea-autocomplete"
   },
   "files": [
-    "index.js",
-    "umd/",
+    "dist",
     "types.js",
     "style.css"
   ],
-  "main": "index.js",
+  "main": "dist/react-textarea-autocomplete.cjs.js",
```

Comment:
Thanks for this. Tbh I was always unsure what are the correct entries what I'm supposed to use. Now I finally know it.

Reply:
😄 I will write a blog post soon about "how to ship libraries nowadays", will try to explain all this crazy stuff, because for such supposedly a simple thing there is a **lot** to know

Reply:
Definitely a good idea. Send me the link afterward. I'd like to read it.

Reply:
@Andarist yeah, please ping me as well once it's written. I'd appreciate something like this.

Reply:
@jukben @jvorcak 
I've published first part
https://developers.livechatinc.com/blog/how-to-create-javascript-libraries-in-2018-part-1/

Enjoy 😃

Reply:
@jukben @jvorcak I've published 2nd part if you are interested https://twitter.com/AndaristRake/status/971376983841046562 . It's even mentioning this repo 😉